templates/documentation/datasheet-documentation.html

```diff
@@ -0,0 +1,243 @@
+                                        <li class="py-1">
+                                            Admin will be able to upload a preset CSV of preferences. This will prevent
+                                            admin from having to manually input respective data one by one. Admin will
+                                            see an error message if CSV is in the incorrect format. Follow the
+                                            instructions given in the message and reupload file
+                                        </li>
+                                        <li class="py-1">
+                                            To upload a new file, click [Upload New File]
+                                        </li>
+                                        <li class="py-1">
+                                            Select corresponding category. Click on [Choose File] and select the
+                                            corresponding file to its category
+                                        </li>
+                                    </ul>
+                                    <div class="cursor-default"
+                                         @click="selected !== 3 ? selected = 3 : selected = null">
+                                        <span>
+                                            [
+                                        </span>
+                                        <span class="hover:underline text-blue-500">
+                                            close
+                                        </span>
+                                        <span>
+                                            ]
+                                        </span>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </li>
+                    <li class="relative border-b border-gray-200">
+                        <button type="button" class="w-full px-8 py-6 text-left"
+                                @click="selected !== 4 ? selected = 4 : selected = null">
+                            <div class="text-2xl flex items-center justify-between">
+                                <span>Manage Data</span>
+                                <span class="ico-plus"></span>
+                            </div>
+                        </button>
+                        <div id = "select4" class="relative overflow-hidden transition-all max-h-0 duration-700" style=""
+                             x-ref="container4"
+                             x-bind:style="selected == 4 ? 'max-height: ' + $refs.container4.scrollHeight + 'px' : ''">
+
+                            <div class="text-left items-center">
+                                <div class="p-6">
+                                    <ul class="list-disc pl-8">
+                                        <img src="{% static 'images/ds4.gif' %}"
```

Comment:
this image is barely visible - the resolution is super low. Also why is this a gif? Isn't it a static image?

Reply:
In general many of the images are super low resolution

Reply:
Yeah looking through the pages, the screenshots are too low resolution for them to communicate the information they carry